src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1442,6 +1442,10 @@
     <Resource Include="UI\Images\profile_normal.png" />
   </ItemGroup>
   <ItemGroup>
+    <EmbeddedResource Include="Views\GuidedTour\HtmlPages\Resources\filterIcon.png" />
+    <EmbeddedResource Include="Views\GuidedTour\HtmlPages\Resources\sortIcon.png" />
```

Comment:
Hi @filipeotero , would you check if the filter and sort icons are already part of the project?

Reply:
Hi @QilongTang, I just found the icons with transparent backgrounds.

Reply:
Oh I was not aware that transparent  background not acceptable?

Reply:
It's acceptable but I will need to adjust the html haha.
Another thing is that we only search the resources inside _'Views\GuidedTour\HtmlPages\Resources\'_  folder :slightly_frowning_face:

Reply:
Make sense, enough difference to keep this version